docs/installation/installing.rst

```diff
@@ -23,6 +18,13 @@ Docker support
     :local:
 
 
+---------------------------------------
```

Comment:
this heading shouldn't be at the same level as the other headings in this topic (i.e. it's not as important as the other content). it seems the best way to do this is to make it a definition list, e.g.:

```
Supported Versions and Interoperability
    .. include:: ../includes/ksql-supported-versions.rst
```